app/src/main/res/layout/activity_profile.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we go compose only as mentioned before? :)

Reply:
handled orientation changes in compose in profile-section